crates/router/src/connector/aci/transformers.rs

```diff
@@ -111,6 +111,11 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for AciPaymentsRequest {
             api::PaymentMethodData::PayLater(_) => PaymentDetails::Klarna,
             api::PaymentMethodData::Wallet(_) => PaymentDetails::Wallet,
             api::PaymentMethodData::BankRedirect(_) => PaymentDetails::BankRedirect,
+            _ => Err(errors::ConnectorError::NotSupported {
```

Comment:
Avoid using underscore to handle other cases, using underscore we can find `NotSupported` only in runtime and not in compile time.

Reply:
Add Crypto variant here